gettingStarted/sysElevenStackKickstart.yaml

```diff
@@ -141,13 +162,15 @@ resources:
   net:
     type: OS::Neutron::Net
     properties: 
-      name: kickstart-net
+      name: 
+        list_join: [ '-', [ { get_param: instance_name }, 'net']]
 
   subnet:
     type: OS::Neutron::Subnet
     depends_on: [ net ]
     properties:
-      name: kickstart-subnet
+      name: 
+        list_join: [ '-', [ { get_param: instance_name }, 'subnet']]
```

Comment:
would be nice if these resources would be called `$stackname-kickstart-subnet` etc.